Assets/MRTK/Core/Providers/BaseInputDeviceManager.cs

```diff
@@ -150,11 +148,15 @@ public override void Initialize()
                     pointerConfigurations[i].cache = new Stack<IMixedRealityPointer>();
                 }
             }
+
+            base.Initialize();
```

Comment:
Why the move to put the `base` call at the end? Could this lean to a change in behavior if derived classes expect these to be called in a specific order?

Reply:
the thought was to ensure that derived class initialization succeeds before the base class sets the IsInitialized flag. happy to move it back.

Reply:
revisited these changes.